admin/test/TestAppLoader.scala

```diff
@@ -11,5 +20,12 @@ trait TestComponents {
 
 class TestAppLoader extends AppLoader {
   override def buildComponents(context: Context): FrontendComponents =
-    new BuiltInComponentsFromContext(context) with AppComponents with TestComponents
+    new BuiltInComponentsFromContext(context) with AppComponents with TestComponents with MockitoSugar {
+      private val mockAuth = mock[GuardianAuthWithExemptions]
+      when(mockAuth.filter).thenReturn(new Filter {
+        implicit val mat: Materializer = materializer
+        def apply(nextFilter: RequestHeader => Future[Result])(request: RequestHeader) = nextFilter(request) // no auth
```

Comment:
What does `no auth` mean here? Are we passing an unauthenticated request to the next filter?

Reply:
oh it just means in the test we're essentially saying the filter does nothing, so essentially 'yes' to your question

the tests that this change gets working, weren't testing auth - hence I think this is justified